.github/CONTRIBUTING.md

```diff
@@ -69,102 +67,142 @@ There are few ways you can play with the build:
 -  Playing with `webchat-loader`
    -  Navigate to https://compulim.github.io/webchat-loader/
    -  In the version dropdown, select `http://localhost:5000/webchat-es5.js`
--  Write your own web app
+-  Write your own HTML page to load Web Chat
    -  `<script src="http://localhost:5000/webchat.js"></script>`
-   -  Or using tarballs
+-  Create a new React app and symlink or load tarballs from these packages, in this specific order
+   1. `/packages/core`
+   1. `/packages/api`
+   1. `/packages/component`
+   1. `/packages/directlinespeech`
+   1. `/packages/bundle`
+
+# How to contribute to our code
 
-# Readying for publication
+1. [Write test pages first](#test-driven-development)
+1. Fix the issue or implement the new feature
+1. [Run static code analysis](#static-code-analysis)
+1. [Final checks](#final-checks)
+
+## Test driven development
 
 We care about software quality. Quality checking prevents regressions, reduces maintenance costs, minimizes chores, and enables us to move faster.
 
-Before submitting the pull request, please check your build by running these checks:
+For bugs, write test page to reproduce the bug, then fix it. This will prevent future regression.
 
--  [Running automated tests](#running-automated-tests)
--  [Performing static code analysis](#static-code-analysis)
+For features, use test page as a playground. Write new tests to verify different sub-features, e.g. rendering in carousel layout vs. rendering in stacked layout. Also write tests for both happy and unhappy paths. This will future-proof the work and protect the investment.
```

Comment:
```suggestion
For features, use the test page as a playground. Write new tests to verify different sub-features, e.g. rendering in carousel layout vs. rendering in stacked layout. Also write tests for both happy and unhappy paths. This will future-proof the work and protect the investment.
```

Reply:
I feel like these directions are contradictory to the suggestions of using playground/loader above

Reply:
Right, the approach is confusing.

Test page is for developer who is developing the feature, so they can do TDD.

Playground is for developer who do not own that feature, but would like to try it out.

What do you think?

Reply:
I see, makes total sense, haha. Yeah, could you add a comment to that effect in both sections?

Playground: 'Using the Playground (and loader) is a great way to test out the latest from Web Chat'

Test page: 'Using a test page is recommended over using the playground or loader for development, for faster loading and quick reproduce ability. '

Reply:
I separated the bullets (above, around line 63) into 2 sections: "Our prebuilt environments" and "Your own HTML page".

What do you think?

Reply:
looks good, ty!